pkg/pillar/types/contenttreetypes.go

```diff
@@ -131,6 +132,8 @@ type ContentTreeStatus struct {
 	NameIsURL    bool
 	// Blobs the sha256 hashes of the blobs that are in this tree, the first of which always is the root
 	Blobs []string
+	// kube oci image
```

Comment:
This comment doesn't tell me what it is.

Reply:
Will add comments.